astropix.py

```diff
@@ -270,18 +281,21 @@ def init_injection(self, slot: int = 3, inj_voltage:float = None, inj_period:int
         pulseperset: int
         dac_config:tuple[int, list[float]]: injdac settings. Must be fully specified if set. 
         """
-        default_injdac = (2, [0.4, 0.0])
+        # Default configuration for the dac
+        default_injdac = (2, [0.4, 0.3])
```

Comment:
I misspoke here, I'm sorry. Injection is the first element of the input array. Default settings should be `default_injdac = (2, [0.3, 0.0])`